gbfs.md

```diff
@@ -576,10 +578,83 @@ Field Name | Required | Type | Defines
+\-&nbsp;`per_min_pricing` | Optional | Array | Array of segments when the price is a function of time travelled, displayed in minutes.<br /><br />Total price is the addition of `price` and all segments in `per_km_pricing` and `per_min_pricing`. If this array is not provided, there are no variable prices based on time.
+&emsp;&emsp;\-&nbsp;`start` | Yes | Non-Negative Integer | Number of minutes that have to elapse before this segment starts applying.
+&emsp;&emsp;\-&nbsp;`rate` | Yes | Float | Rate that is charged for each minute `interval` after the `start`. Can be a negative number, which indicates that the traveller will receive a discount.
+&emsp;&emsp;\-&nbsp;`interval` | Yes | Non-Negative Integer | Interval in minutes at which the `rate` of this segment is either reapplied indefinitely, or if defined, up until (but not including) `end` minute.<br /><br />An interval of 0 indicates the rate is only charged once.
+&emsp;&emsp;\-&nbsp; `end` | Yes | Non-Negative Integer | The minute at which the rate will no longer apply.<br /><br />If this field is empty, the price issued for this segment is charged until the trip ends, in addition to following segments. 
+\-&nbsp;`surge_pricing` | Optional | Boolean | Is there currently an increase in price in response to increased demand in this pricing plan? If this field is empty, it means these is no surge pricing in effect.<br /><br />`true` - Surge pricing is in effect.<br />  `false` - Surge pricing is not in effect. 
 
+
+Example:
+
+The user does not pay more than the base price for the first 10 km. After 10 km the user pays $1 per km. After 25 km the user pays $0.50 per km and an additional $3 every 5 km, the extension price, in addition to $0.50 per km. 
+
+```jsonc
+{
+  "plans": {
+    "plan_id": "plan2",
+    "vehicle_type_id": "bike1",
+    "name": "One-Way",
+    "currency": "USD",
+    "price": 2,
+    "is_taxable": false,
+    "description": "Includes 10km, overage fees apply after 10km.",
+    "per_km_pricing": [
+      {
+        "start": 10,
+        "rate": 1,
+        "interval": 1,
+        "end": 25
+      }, 
+      {
+        "start": 25,
+        "rate": 0.5,
+        "interval": 1
+      },
+      {
+        "start": 25,
+        "rate": 3,
+        "interval": 5
+      }
+    ]
+  }
+}
+```
+Example:
+
+This example demonstrates a pricing scheme that has a rate both by minute and by km. The user is charged $0.25 per km as well as $0.50 per minute. Both of these rates happen concurrently and are not dependent on one another. 
+```jsonc
+{
+  "plans": {
+    "plan_id": "plan3",
+    "vehicle_type_id": "scooter1",
+    "name": "Simple Rate",
+    "currency": "CAD",
+    "price": 3,
+    "is_taxable": true,
+    "description": "$3 unlock fee, $0.25 per kilometer and 0.50 per minute.",
+    "per_km_pricing": {
+      "start": 0,
+      "rate": 0.25,
+      "interval": 1,
+    },
+    "per_min_pricing": {
+      "start": 0,
+      "rate": 0.50,
+      "interval": 1
+    }
```

Comment:
nit:  `per_km_pricing ` and `per_min_pricing ` should be array

Reply:
Thanks! We've fixed it.